### DISCUSSION TITLE
Clear scrollback each refresh or always retain dividing line in scrollback

### DISCUSSION THREAD

**Discussion by bpn19** [leo-arch/clifm#231] [Q&A] [answered]
I've been using this program for quite some time now. Generally have no issues with it, but one thing that I've found really bothersome is that I will often do many consecutive operations, each followed by a refresh (i.e. 'rf' or enter). But if I scroll back, there is no dividing line separating the directory's current content from the previous content. Not such a big deal if the current content fits entirely within the current display, but if there are many files, then the scrollback buffer is needed (I do not use the pager, nor want to).

So is there a way to either:
1. Clear scrollback after every refresh, or;
2. Ensure the scrollback always retains a dividing line

Seems like there should ideally be config file options for both. I use kitty as my terminal.

**Comment by bpn19** [comment, marked as answer]
So I was looking in the source code a little bit and I realised that there was no reason for things to be missing in the scrollback. I had set 'ClearScreen=true' in the config at some point. Changing that to false, I now get everything including the prompt and dividing line in the scrollback. Closing as resolved.

**Reply by leo-arch** [reply]
I'm glad the issue is solved @bpn19. Please do not hesitate to open a new issue is something else comes out.
